### PR TITLE
Feature/utils

### PR DESCRIPTION
**Pull request by sanjitsarkar**
added more box-shadow classes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,239 +1,17 @@
...
-        <svg></svg>    
-          
-        lorem ipsum dolor sit amet
-        <span class="alert-close" onclick="closeAlert(this)">&times;</span>
-      
-      </div>
-  </section>
-
-  <section class="avatar-section row flex-gap-1 mt-1 align-center ">
-    <!-- <div class="avatar-container">
```
Comment (Megha-Joshi) [inline index.html:47]: Remove the comments

File: index.html [hunk 2]
```diff
@@ -1,239 +1,17 @@
...
-   <div class="card-footer">
-     <div class="row items-center flex-gap-05">
-       <h4 class="o-70 text-600">Price</h4>
-       <h4 class="text-base text-md-bold">₹10200000</h4>
-     </div>
-     <div class="card-actions">
-       <button class="btn btn-round-md btn-dark">
-      <i class="fa fa-shopping-cart"></i>
-         
-       </button>
```
Comment (Megha-Joshi) [inline index.html:152]: Remove the undesired space between tags.

File: index.html [hunk 3]
```diff
@@ -1,239 +1,17 @@
...
-     </div>
-    </div>
-   </div>
-    </div>
-    <div class="card w-80">
-    <div class="card-header">
-       
-        <img src="https://images.samsung.com/uk/smartphones/galaxy-s20/images/kv/galaxy-s20_highlights_kv_00.jpg" alt="" srcset="" class="h-40 w-full">
```
Comment (Megha-Joshi) [inline index.html:168]: give some name in alt

**Review by Megha-Joshi — COMMENTED** [review]
Just do the suggested changes otherwise good to merge.

**Review by anshulraheja — COMMENTED** [review]
Everything looks good, you can merge!

**Pull request merged**